component/http/v2/router/httprouter/router_option.go

```diff
@@ -63,16 +63,16 @@ func Middlewares(mm ...middleware.Func) OptionFunc {
 	}
 }
 
-// EnableExpVarProfiling option for enabling expVar in profiling endpoints.
-func EnableExpVarProfiling() OptionFunc {
+// WithEnableExpVarProfiling option for enabling expVar in profiling endpoints.
+func WithEnableExpVarProfiling() OptionFunc {
 	return func(cfg *Config) error {
 		cfg.enableProfilingExpVar = true
 		return nil
 	}
 }
 
-// EnableAppNameHeaders option for adding name and version header to the response.
-func EnableAppNameHeaders(name, version string) OptionFunc {
+// WithEnableAppNameHeaders option for adding name and version header to the response.
+func WithEnableAppNameHeaders(name, version string) OptionFunc {
```

Comment:
I guess `Enable` is redundant here. What about
- `WithExpVarProfiling()`
- `WithAppNameHeaders()`